packages/common/src/intl/locales/en/common.json

```diff
@@ -75,6 +75,8 @@
   "label.expiry": "Expiry",
   "label.general": "General",
   "label.goods-receipt": "Goods receipt",
+  "label.requisition-number": "Reqnum",

```

Comment:
I hadn't actually noticed `Invnum` until now 🙄 
Could possibly just have `Number` or `Req #` ...though now is probably not the time to worry about column names. I'm sure there will be opinions available if we want some 😄